File: samples/ObjC Collection View Inbox/watchkitapp Extension/InterfaceController.m
/* 
 * Licensed Materials - Property of IBM 
 * 
 * 5725E28, 5725I03 
 * 
 * © Copyright IBM Corp. 2017, 2018
 * US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp. 
 */

#import "InterfaceController.h"
#import <IBMMobilePushWatch/IBMMobilePushWatch.h>

@interface InterfaceController ()

@end


@implementation InterfaceController

- (void)awakeWithContext:(id)context {
    [self.versionLabel setText: MCEWatchSdk.sharedInstance.sdkVersion];
    [super awakeWithContext:context];
}

@end

File: samples/Swift Combined Inbox Sample/SampleWatch Extension/InterfaceController.swift
/*
 * Licensed Materials - Property of IBM
 *
 * 5725E28, 5725I03
 *
 * © Copyright IBM Corp. 2017, 2018
 * US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
 */

import WatchKit
import Foundation

class InterfaceController: WKInterfaceController {
    @IBOutlet weak var versionLabel: WKInterfaceLabel?

    override func awake(withContext context: Any?) {
        super.awake(withContext: context)
        versionLabel?.setText(MCEWatchSdk.sharedInstance().sdkVersion())
    }
}
